Docker/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get update && \
 RUN mkdir -p /var/www/FreshRSS/ /run/apache2/
 WORKDIR /var/www/FreshRSS
 
-COPY . /var/www/FreshRSS
+COPY --chown=:www-data . /var/www/FreshRSS
```

Comment:
For some reason, this syntax fails on my Docker `Docker version 27.5.1, build 27.5.1-0ubuntu3` with the following error message:
> unable to convert uid/gid chown string to host mapping: can't find uid for user : no such user:

Reply:
Strange, I checked the docs and there's no mention of the user-less syntax that I used but it seemed to work. Your version is definitely correct though.